engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -1668,7 +1746,7 @@ abstract class SemanticsUpdateBuilder {
   ///    `validationResult` argument.
   void updateNode({
     required int id,
-    required int flags,
+    required List<bool> flags,
```

Comment:
Can we use NativeFieldWrapperClass1 for SemanticsFlags so that we don't do this serialization?

See StringAttribute class for example